### PR TITLE
Updated project.md

### PR DESCRIPTION
**Pull request by KaKashi1210**
Nitpicking - fixed some typos and improved some sentences.

pr addressing issue - #2508

please let me know if you want more changes.

### CODE REVIEW

**Comment by KaKashi1210** [comment]
@susanev can you please take a look.

**Comment by susanev** [comment]
thank you @KaKashi1210!! im technically on holiday this week, so will take a look first thing next monday morning.

**Comment by KaKashi1210** [comment]
Ohh my bad, have a good one!

File: themes/default/content/docs/reference/project-file.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ plugins:
 | - | - | - | - |
 | `name` | required | Name of the project containing alphanumeric characters, hyphens, underscores, and periods. | None. |
 | `runtime` | required | Installed language runtime of the project: `nodejs`, `python`, `go`, `dotnet`, `java` or `yaml`. | [runtime options](#runtime-options)
-| `description` | optional | Description of the project. | None. |
+| `description` | optional | Description of the project (A breif description of project). | None. |
```
Comment (susanev) [inline themes/default/content/docs/reference/project-file.md:61]:
```suggestion
| `description` | optional | A brief description of the project. | None. |
```

**Review by susanev — APPROVED** [review]
sry this took me sooo long to get to; thx for fixing this up!! 🙌🏻

**Comment by susanev** [comment]
@sean1588 can you help shepherd this in?

**Comment by sean1588** [comment]
@susanev yeah I can shephard it in

**Pull request merged**